Jellyfin.HardwareVisualizer/Server/Database/MediaTestFile.cs

```diff
@@ -11,7 +11,7 @@ public class MediaTestFile : HashedExternalEntity
 
 	public long Bitrate { get; set; }
 	public long Size { get; set; }
-	public string VideoCodec { get; set; }
+	public Guid VideoCodec { get; set; }
 	public string AudioCodec { get; set; }
 
```

Comment:
Missing EF navigation property

Reply:
So essentially from what I have seen its something like `ICollection<MediaTestFile> VideoCodec` or something like that?